src/app/customer/Customer.tsx

```diff
@@ -263,11 +307,16 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
 
         const email = formValues.email.trim();
         try {
-            const { data } = await continueAsGuest({
+            const credentials = {
                 email,
                 acceptsMarketingNewsletter: canSubscribe && formValues.shouldSubscribe ? true : undefined,
                 acceptsAbandonedCartEmails: formValues.shouldSubscribe ? true : undefined,
-            });
+            };
+
+            const { data } = await continueAsGuest(
+                credentials,
+                { methodId: this.providerIdWithCustomContinueFlow }
```

Comment:
Correct me if I'm wrong, if Bolt is enabled for the store, shoppers must use Bolt - there's no choice not to use it? Why do we want to remove that choice. And secondly why does it take priorities over other wallet payment methods that we offer, such as GooglePay, where we offer a choice for shoppers to choose?

cc @bc-charlesho

Reply:
@davidchin shopper can close Bolt modal and continue with default flow.

I can't answer on the second question, @BC-TKozub can you help me with the second one?

fyi @BC-ACherednichenko

Reply:
@davidchin Only shoppers who have Bolt account will see a pop up on checkout.

Reply:
Ok thank you. That makes it _slightly_ better. But there's still a concern where Bolt has priority over other wallet methods as well as interfering with our own guest checkout flow where people are prompted to sign into their BC account if they have one. Anyway this is ultimately a Product question so I'll let Charles to get an alignment with his team.